pandas/core/groupby/indexing.py

```diff
@@ -0,0 +1,262 @@
+from __future__ import annotations
+
+import numpy as np
+
+from pandas.util._decorators import doc
+
+from pandas.core.groupby import groupby
+from pandas.core.indexes.api import CategoricalIndex
+
+
+class GroupByIndexingMixin:
+    """
+    Mixin for adding .rows to GroupBy.
+    """
+
+    @property
+    def rows(self) -> _rowsGroupByIndexer:
```

Comment:
make this private for now (leave the doc-string pending UX discussions)

Reply:
Done

Reply:
umm `_rows`